pkg/virt-handler/isolation/validation.go

```diff
@@ -24,12 +24,13 @@ func GetImageInfo(imagePath string, context IsolationResult, config *v1.DiskVeri
 		QEMUIMGPath, "info", imagePath, "--output", "json",
 	).Output()
 	if err != nil {
+		memError := fmt.Errorf("trying to increase the verification memory usage limit by specifying the DiskVerification field in the developer configuration options in the KubeVirt CR might solve this")
 		if e, ok := err.(*exec.ExitError); ok {
 			if len(e.Stderr) > 0 {
-				return nil, fmt.Errorf("failed to invoke qemu-img: %v: '%v'", err, string(e.Stderr))
+				return nil, fmt.Errorf("failed to invoke qemu-img: %v: '%v', %v", err, string(e.Stderr), memError)
```

Comment:
I see this is being returned on every crash, can we somehow check that the crash was due to memory error? not sure if it's possible though

Reply:
I don't think we can, being memory-constrained can cause different types of failures, so I wouldn't know how to do it.